src/ViewshedInTabletopAR/FormsDemoAR/FormsDemoAR/ARPage.xaml.cs

```diff
@@ -0,0 +1,231 @@
+﻿using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.GeoAnalysis;
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Timers;
+using Xamarin.Forms;
+
+namespace FormsDemoAR
+{
+    // Learn more about making custom code visible in the Xamarin.Forms previewer
+    // by visiting https://aka.ms/xamarinforms-previewer
+    [DesignTimeVisible(false)]
+    public partial class ARPage : ContentPage
+    {
+        private MapPoint centerPoint;
+
+        private Graphic planeGraphic;
+
+        Polyline routePath;
+        double routeLength;
+        double progressOnRoute = 0;
+
+        public ARPage()
+        {
+            InitializeComponent();
+        }
+
+        private async void InitializeScene()
+        {
+            try
+            {
+                // Set the translation factor to allow you to see the whole scene
+                // TranslationFactor = scene width / physical table width
+                arSceneView.TranslationFactor = 4440.0 / .7;
+
+                // Disable touch interaction
+                arSceneView.InteractionOptions = new SceneViewInteractionOptions { IsEnabled = false };
+
+                // Set up the scene
+                await configureScene();
+
+                // Add the airplane graphic
+                await createAndAddAirplane();
+
+                buildFlightPath();
+
+                // Animate the airplane
+                animatePlane();
+
+                // Show the analysis
+                showViewshed();
+            }
+            catch { }
+        }
+
+        private async Task configureScene()
+        {
+            // Create a scene and add the Melbourne layer to it.
+            Scene melbourneScene = new Scene(SceneViewTilingScheme.Geographic);
```

Comment:
Don't provide the tilingscheme (this one won't work with most tiled services either). Better to go with default